pkg/controller/logstorage/linseed/linseed_controller.go

```diff
@@ -136,6 +136,7 @@ func Add(mgr manager.Manager, opts options.AddOptions) error {
 		render.LinseedTokenSecret,
 		monitor.PrometheusClientTLSSecretName,
 		render.ElasticsearchLinseedUserSecret,
+		render.ElasticsearchLinseedUserSecret,
```

Comment:
These look the same, is that expected?

Reply:
I just realized I hadn't commented on that, yeah I agree. We shouldn't need that repeated here.